app/dashboards/work_process_dashboard.rb

```diff
@@ -40,18 +40,16 @@ class WorkProcessDashboard < Administrate::BaseDashboard
   ].freeze
 
   FORM_ATTRIBUTES = %i[
-    competencies
-    default_hours
+    title
     description
+    default_hours
     maximum_hours
     minimum_hours
-    sort_order
-    title
   ].freeze
```

Comment:
This change is affecting the existing edit page for the work processes. It's interesting to note that the current behavior takes forever to load because of the competencies.

Reply:
@clarissalimab can you add a new task to not forget about this? thanks!

Reply:
Before the change:
<img width="1053" alt="image" src="https://github.com/user-attachments/assets/5af6b4d5-ccaa-4884-8912-49c76fe09235" />

After the change:
<img width="1048" alt="image" src="https://github.com/user-attachments/assets/076a670a-6b11-45c5-8474-d2cad0c392ef" />

Reply:
This is resolved as I merged #1006 into this PR.